tests/framework/CodeInterfaceTests/Neutrino/gold/Neutrino_Base/sampleNeutrino/3/results.csv

```diff
@@ -0,0 +1,302 @@
+time,result

```

Comment:
you should remove this... you are not testing against this... you just test _"Neutrino_Base/out_dump.csv"_ in your **tests** file